### DISCUSSION TITLE
Request for Laboratory Flask Icon

### DISCUSSION THREAD

**Discussion by claitonllemes** [tabler/tabler-icons#732] [Ideas]
Laboratory Flask Icon I didn't find anything like that by searching for lab, bottle, jar, etc.

![download](https://github.com/tabler/tabler-icons/assets/99222756/80b55e33-0d43-4dbc-bdbd-8fe1ce2c1020)
![images](https://github.com/tabler/tabler-icons/assets/99222756/22299a95-c6c9-49fd-9ad7-d96a02e29825)
![images-3](https://github.com/tabler/tabler-icons/assets/99222756/ef15f174-3ec8-4547-95c6-34b564494173)
![images-2](https://github.com/tabler/tabler-icons/assets/99222756/30078b66-d0a1-4c39-82cd-c0f683061760)
![images-1](https://github.com/tabler/tabler-icons/assets/99222756/09ba708d-ad2f-4d7a-9070-dbe99bacaaf3)

**Comment by divdax** [comment]
There is `flask` and `flask-2`.
